### PR TITLE
[JBIDE-20031] Filter imported projects by gitcontextdir

### PR DESCRIPTION
**Pull request by fbricon**
Signed-off-by: Fred Bricon <fbricon@gmail.com>

### CODE REVIEW

File: plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/importapp/operation/ImportNewProject.java [hunk 1]
```diff
@@ -155,4 +164,22 @@ protected boolean cloneDestinationExists() {
 		return cloneDestination != null
 				&& cloneDestination.exists();
 	}
+
+	private static Collection<String> sanitize(Collection<String> filters) {
+		Collection<String> sanitized = null;
+		if (filters != null) {
+			sanitized = new LinkedHashSet<>(filters.size());
+			for (String path : filters) {
+				if (!StringUtils.isEmptyOrNull(path)) {
```
Comment (jcantrill) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/importapp/operation/ImportNewProject.java]: We have Apache commons StringUtils available which provides things like 'isBlank' which handles this case

**Comment by jcantrill** [comment]
LGTM

File: plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/importapp/operation/ImportNewProject.java (hunk 1 above)
Comment (fbricon) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/importapp/operation/ImportNewProject.java]: Yes, several StringUtils libs are available in the classpath. I chose the one from the project

**Comment by fbricon** [comment]
Applied in master

**Pull request closed without merging**